### ISSUE TITLE
Spyder GUI crashes after upgrade to Ubuntu 18.10

### ISSUE THREAD

**Issue by shihabkhan1** [spyder-ide/spyder#8177]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
When I open spyder, the GUI crashes after a while. I have to do ctrl+c in the terminal to completely stop the spyder process.

### What steps reproduce the problem?

1. Opening and using Spyder GUI.

### What is the expected output? What do you see instead?
The GUI crashes. This problem started after I upgraded to Ubuntu 18.10. After the upgrade I wasn't able to boot up. I read on forums that this issue could be related to my Graphics drivers. Hence I removed NVIDIA drivers and then I was able to boot up. But then Spyder started giving me this issue.

### Paste Traceback/Error Below (if applicable)

I see the following things on my terminal while using Spyder GUI.
```python-traceback

nouveau: kernel rejected pushbuf: No such file or directory
nouveau: ch5: krec 0 pushes 0 bufs 1 relocs 0
nouveau: ch5: buf 00000000 00000002 00000004 00000004 00000000
nouveau: kernel rejected pushbuf: No such file or directory
nouveau: ch5: krec 0 pushes 0 bufs 2 relocs 0
nouveau: ch5: buf 00000000 00000002 00000004 00000004 00000000
nouveau: ch5: buf 00000001 00000006 00000004 00000000 00000004
nouveau: kernel rejected pushbuf: No such file or directory
nouveau: ch5: krec 0 pushes 0 bufs 1 relocs 0
nouveau: ch5: buf 00000000 00000003 00000004 00000004 00000000
nouveau: kernel rejected pushbuf: No such file or directory
nouveau: ch5: krec 0 pushes 0 bufs 1 relocs 0
nouveau: ch5: buf 00000000 00000003 00000004 00000004 00000000
python: ../nouveau/pushbuf.c:723: nouveau_pushbuf_data: Assertion `kref' failed.

```

## Versions

* Spyder version: Spyder 3.3.1 
* Python version:  3.6.6 
* Qt version: 5.9.6
* PyQt version: 5.9.2
* Operating System name/version: Linux

**Comment by ccordoba12** [comment]
This is a problem with your graphics card drivers. For now, please use your Intel card by default instead of the Nvidia one.

This could be solved in our next release (3.3.2, to be released next week), which will use software rendering instead of accessing the graphics card. But in case it doesn't, there's nothing we can do about it, sorry.

**Comment by shihabkhan1** [comment]
I downgraded PyQt to 5.6 and now spyder is working.
